### ISSUE TITLE
Composer missing 5.8

### ISSUE THREAD

**Issue by KeithBush** [Askedio/laravel-soft-cascade#96]
I'm unable to pull version 5.8.0 via composer

Packagist showing latest version is 5.7.0, repo is showing the latest package as 5.8.0
[https://packagist.org/packages/askedio/laravel-soft-cascade](https://packagist.org/packages/askedio/laravel-soft-cascade)

**Comment by trm42** [comment]
Was going to report the same problem. Composer error message:
```
$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package askedio/laravel-soft-cascade ^5.8.0 exists as askedio/laravel-soft-cascade[1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 5.5.0, 5.5.1, 5.5.10, 5.5.11, 5.5.12, 5.5.13, 5.5.14, 5.5.15, 5.5.2, 5.5.3, 5.5.4, 5.5.5, 5.5.6, 5.5.7, 5.5.8, 5.5.9, 5.5.x-dev, 5.6.0, 5.6.1, 5.6.2, 5.6.3, 5.6.4, 5.6.5, 5.6.x-dev, 5.7.0, 5.7.x-dev, dev-master] but these are rejected by your constraint.
```

**Comment by jakesyl** [comment]
Having the same issue.

**Comment by jakesylvestre** [comment]
I'm having the same issue. Appears to be an [authorization issue](https://github.com/Askedio/laravel-soft-cascade/pull/94#issuecomment-468283145) with GitHub (which is discussed on #94). In the meantime, @exodusanto has put forward a [workaround](https://github.com/Askedio/laravel-soft-cascade/pull/94#issuecomment-468907593) namely, to [add this](https://getcomposer.org/doc/05-repositories.md#packages) to  `repositories` in your `composer.json`

Please update this issue when this issue's resolved. Appreciate it!

**Comment by tswestendorp** [comment]
Looks like this has been fixed, just reinstalled and got 5.8

**Comment by KeithBush** [comment]
Packagist showing 5.8.0 among the available versions now.
https://packagist.org/packages/askedio/laravel-soft-cascade#5.8.0
